e2e/cypress/integration/accessibility/accessibility_dropdowns_spec.js

```diff
@@ -89,7 +89,7 @@ describe('Verify Accessibility Support in Dropdown Menus', () => {
         // * Verify the accessibility support in the Main Menu Dropdown items
         cy.apiGetConfig().then((response) => {
             const siteName = response.body.TeamSettings.SiteName;
-            const labels = ['Account Settings dialog', 'Invite People dialog', 'Team Settings dialog', 'Manage Members dialog', '', '', 'Leave Team dialog', '', 'Plugin Marketplace dialog', '', '', '', '', '', `About ${siteName} dialog`, ''];
+            const labels = ['Account Settings dialog', 'Invite People dialog', 'Team Settings dialog', 'Manage Members dialog', '', 'Leave Team dialog', '', '', 'Plugin Marketplace dialog', '', '', '', '', '', `About ${siteName} dialog`, ''];
```

Comment:
I still got failing test here:
<img width="351" alt="Screen Shot 2020-06-24 at 4 07 48 PM" src="https://user-images.githubusercontent.com/5334504/85520634-c3a34300-b635-11ea-8999-947f06b1fc93.png">

Reply:
There is an extra element added after the 'Leave Team dialog' which is causing this failure. If this is removed the test works fine. Here is what works for me:

`const labels = ['Account Settings dialog', 'Invite People dialog', 'Team Settings dialog', 'Manage Members dialog', '', 'Leave Team dialog', '', 'Plugin Marketplace dialog', '', '', '', '', '', `About ${siteName} dialog`, ''];`

Reply:
@saturninoabril @srkgupta Updated in latest commit. My local had `Incidents & Playbook` plugin so that menu item was appearing in the list.